infra/license/license-checker.ts

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {readFileSync, writeFileSync} from 'fs';
+import {EOL} from 'os';
+
+import * as licenseJudgment from './license-judgment.json';
+import * as packageJudgment from './package-judgment.json';
+
+
+/**
+ * Argument List
+ *  - Input path for list of used licenses
+ *  - Tag name for the verification
+ */
+var args = process.argv.slice(2)
+const inputPathOfLicenseList = args[0];
+const verificationTag = args[1];
+
+/**
+ * - WarningCount : The total number of warnings
+ *
+ * - WarnnedLicenseUsed
+ *   - Licenses which are classified as WARN in license-judgment.json
+ * - NeverChecked
+ *   - Licenses which are not found in license-judgment.json
+ * - UnknownLicense
+ *   - Licenses which package are not found.
+ */
+var warningList = {
+  'WarningCount': 0,
+
+  'WarnedLicenseUsed': [] as string[],
+  'NeverChecked': [] as string[],
+  'UnknownLicense': [] as string[]
+};
+
+/**
+ * - DeniedCount : The total number of denials
+ *
+ * - ONEForbidden
+ *   - Licenses which are classified as not permitted at ONE in package-judgment.json
+ * - DeniedLicenseUsed
+ *   - Licenses which are denied by license-judgment.json
+ */
+var deniedList = {
+  'DeniedCount': 0,
+
+  'ONEForbidden': [] as string[],
+  'DeniedLicenseUsed': [] as string[]
+};
+
+/**
+ * License Verification start
+ */
+const usedLicenseList = JSON.parse(readFileSync(inputPathOfLicenseList, 'utf-8'));
```

Comment:
Could you show some example of `inputPathOfLicenseList`? it's kinda hard to read the code without knowing the schema.

Reply:
Sorry, it's result of `license-checker` 😢 
Example is as following..!

```json
{
  "@babel/code-frame@7.12.11": {
    "licenses": "MIT",
    "repository": "https://github.com/babel/babel",
    "publisher": "Sebastian McKenzie",
    "email": "sebmck@gmail.com",
    "dependencyPath": "/home/seok/HDD/repos/ONE-vscode/node_modules/@babel/code-frame",
    "path": "/home/seok/HDD/repos/ONE-vscode/node_modules/@babel/code-frame",
    "licenseFile": "/home/seok/HDD/repos/ONE-vscode/node_modules/@babel/code-frame/LICENSE"
  },
  "@babel/helper-validator-identifier@7.16.7": {
    "licenses": "MIT",
    "repository": "https://github.com/babel/babel",
    "publisher": "The Babel Team",
    "url": "https://babel.dev/team",
    "dependencyPath": "/home/seok/HDD/repos/ONE-vscode/node_modules/@babel/helper-validator-identifier",
    "path": "/home/seok/HDD/repos/ONE-vscode/node_modules/@babel/helper-validator-identifier",
    "licenseFile": "/home/seok/HDD/repos/ONE-vscode/node_modules/@babel/helper-validator-identifier/LICENSE"
  },
  ...
```

I will add a description about this!